src/web/controls/main-ctrl.js

```diff
@@ -146,6 +188,7 @@ const initialState = {
   wallet: [], // [{name: 'My REV account', ...newRevAddress()}],
   // Dev mode  (show local networks)
   devMode,
+  autoSelectDisabled: autoSelectDisabled
```

Comment:
Please use `autoSelect` as a positive meaning of the boolean field with the default `false` as disabled.